### PR TITLE
INLOG-11 Separate queue and file handlers from configure_logger

### PR DESCRIPTION
**Pull request by pzinemanas**
In this PR the file and queue handlers are separated from the general `configure_logger` function. 

Changes:
- Remove file and queue handlers from `configure_logger`.
- Create a `configure_file_handler` function to append a file handler.
- Create `configure_queue_handler` to create a queue handler.
- Create `clean_ingenia_handlers` to clean ingenia_handlers variable. This is needed to run the tests because the reload function in the fixture is not doing it. 

Tests:
- Unit tests are updated accordingly.

### CODE REVIEW

File: ingenialogger/ingenialogger.py [hunk 1]
```diff
@@ -96,23 +89,49 @@ def configure_logger(level=logging.WARNING, queue=False, file=None):
...
+        log_queue = Queue()
+        queue_handler = QueueHandler(log_queue)
+        queue_handler.setFormatter(formatter)
+        ingenia_handlers.queue_handler = queue_handler
+    if not check_logger_handler(root_logger, ingenia_handlers.queue_handler):
+        root_logger.addHandler(ingenia_handlers.queue_handler)
+    return ingenia_handlers.queue_handler.queue
+
+
+def clean_ingenia_handlers():
```
Comment (GullinBustin) [inline ingenialogger/ingenialogger.py:131]: Like

**Review by GullinBustin — APPROVED** [review]

File: ingenialogger/ingenialogger.py [hunk 2]
```diff
@@ -96,23 +89,49 @@ def configure_logger(level=logging.WARNING, queue=False, file=None):
...
+        ingenia_handlers.file_handler = file_handler
+    if not check_logger_handler(root_logger, ingenia_handlers.file_handler):
+        root_logger.addHandler(ingenia_handlers.file_handler)
+
+
+def configure_queue_handler():
+    """Configure a QueueHandler.
+
+    Returns:
+        queue.Queue: Log queue.
```
Comment (WoodElv) [inline ingenialogger/ingenialogger.py:115]: I think it's like this:
`Queue: Log queue`

File: ingenialogger/ingenialogger.py [hunk 3]
```diff
@@ -96,23 +89,49 @@ def configure_logger(level=logging.WARNING, queue=False, file=None):
...
-            ingenia_handlers.queue_handler = queue_handler
-        if not check_logger_handler(root_logger, ingenia_handlers.queue_handler):
-            root_logger.addHandler(ingenia_handlers.queue_handler)
-        return ingenia_handlers.queue_handler.queue
+
+def configure_file_handler(filename):
+    """Configure a FileHandler.
+
+    Args:
+        filename: Path to the file where the log will be stored.
```
Comment (WoodElv) [inline ingenialogger/ingenialogger.py]: lack of the argument type

**Review by WoodElv — APPROVED** [review]

File: ingenialogger/ingenialogger.py (hunk 2 above)
Comment (GullinBustin) [reply ingenialogger/ingenialogger.py:115]: If only Queue is written, documentation has no link to the python documentation
image

**Pull request merged**